gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/GherkinTestRunner.cs

```diff
@@ -41,8 +41,11 @@ public class GherkinTestRunner
             new Dictionary<string, IgnoreReason>
             {
                 // Add here the name of scenarios to ignore and the reason, e.g.:
-                { "g_V_group_byXageX", IgnoreReason.NullKeysInMapNotSupported }
-                //{ "g_V_properties_propertiesXstartTimeX_drop", IgnoreReason.NoReason },
+                { "g_V_group_byXageX", IgnoreReason.NullKeysInMapNotSupported },
+                { "g_withSackX0X_V_outE_sackXsumX_byXweightX_inV_sack_sum", IgnoreReason.NoReason },
```

Comment:
Maybe add an `IgnoreReason` that explains that the assertions don't work here? You could also add a link to the ticket as a comment to make this clear.

Reply:
since the expectation is that we will fix the test suite before release, i've just added a comment to include the ticket.